apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -109,11 +115,20 @@ defmodule SiteWeb.ScheduleController.TimetableController do
   defp all_stops(conn, _) do
     # we override the default fetch of all_stops to not use the date. We will
     # use the date to fetch the actual schedule data.
-    all_stops =
+    all_stops = if conn.assigns.route.id == "Orange" do
+
+      if conn.assigns.direction_id == 0 do
+        stop_ids = ["place-forhl","place-rugg", "place-bbsta", "place-sstat"]
```

Comment:
I think this is backwards too. The timestamps should be increasing as we go down the column:

![image](https://user-images.githubusercontent.com/2136286/185246749-7fdc2650-5cd6-49a3-a252-e94b2203c2c6.png)